src/PowerShellEditorServices.Protocol/Server/DebugAdapter.cs

```diff
@@ -470,22 +470,35 @@ protected void Stop()
             // InitializedEvent will be sent as soon as the RunspaceChanged
             // event gets fired with the attached runspace.
 
-            var runspaceId = 1;
-            if (!int.TryParse(attachParams.RunspaceId, out runspaceId) || runspaceId <= 0)
+            string debugRunspaceCmd;
+            if(attachParams.RunspaceName != null)
```

Comment:
```suggestion
            if (attachParams.RunspaceName != null)
```

Reply:
Nice to see someone as stylistically picky as me.  ;-)